e2e/test/compatibility.cy.spec.js

```diff
@@ -116,17 +116,53 @@ describe("Embedding SDK: shoppy compatibility", () => {
     })
   })
 
-  it("should open user created dashboards", () => {
+  it("should open user created collection", () => {
     cy.visit({
       url: "/admin",
     })
 
     cy.get("#metabase-sdk-root").within(() => {
       cy.findByText("Saved explorations", { timeout: TIMEOUT }).click()
 
-      cy.findByText("User-Generated", { timeout: TIMEOUT }).click()
+      expect(
+        cy.findByText("User-Generated", { timeout: TIMEOUT }).should("exist"),
+      )
+    })
+  })
+
+  it("should display dashboards with sandboxing for different shops", () => {
+    const getOrdersCountForShop = (site) => {
+      cy.findByTestId(`site-switcher-button-${site}`).click()
+
+      return cy
+        .findAllByTestId("scalar-title", { timeout: TIMEOUT })
+        .filter((_, element) => element.textContent.trim() === "Total Orders")
+        .prev('[data-testid="scalar-container"]')
+        .invoke("text")
+    }
+
+    cy.visit("/admin/analytics")
+
+    cy.get("main").within(() => {
+      cy.findByText("Orders", { timeout: TIMEOUT }).click()
+    })
+
+    const shops = ["proficiency", "stitch", "luminara", "pug"]
+    const counts = []
+
+    cy.wrap(shops).each((site) => {
+      getOrdersCountForShop(site).then((text) => {
+        counts.push(text)
+      })
+    })
 
-      expect(cy.findByText("This collection is empty").should("exist"))
+    cy.then(() => {
+      const unique = new Set(counts)
+
+      expect(
+        unique.size,
+        `All counts: [${counts.join(", ")}]`,
+      ).to.be.greaterThan(1)
```

Comment:
In theory, random seeding can generate an equal number of orders for all four shops (405 for each). 

Should we care about it? I assume the chance is quite low.

As an easy solution, I can set the number of seeded shops to `(4 * some value) - 1` that guarantees that at least one shop will have a different number of orders.

Reply:
> As an easy solution, I can set the number of seeded shops to (4 * some value) - 1 that guarantees that at least one shop will have a different number of orders.

That sounds good! If it's not tricky let's do it